app/demo-interactor/PhysicsArrayCalculator.i.hh

```diff
@@ -16,6 +16,15 @@ namespace celeritas
  * Calculate the cross section.
  *
  * Assumes that the energy grid has the same units as particle.energy.
+ *
+ * XXX also this breaks if prime_energy != 1, since the
```

Comment:
What is XXX a standin for?

Reply:
Sorry, vim highlights it like it does the word `TODO` or `FIXME` -- just some text to get attention. I've got a fix on the branch where I move this to the main `src/` directory and will remove the text when I do that. Or I could change this to an official `\todo`.